package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- Update your `scripts` section accordingly
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
}
```

Reply:
Thank you. I have not seen those commands in any of the lessons.